client/verta/verta/_internal_utils/_utils.py

```diff
@@ -784,7 +784,8 @@ def to_builtin(obj):
     if obj_class == "Series":
         return obj.values.tolist()
     if obj_class == "DataFrame":
-        return obj.values.tolist()
+        return list(map(to_builtin, obj.to_dict("records")))
```

Comment:
This seems different from what we discussed, no?

Reply:
So what we want is `[{"feature-1": 1, "feature-2": 2}, {"feature-1": 3, "feature-2": 4}]`, right? 🤔

Reply:
lmk if you have sth else in mind.

Reply:
Oh I think what you want is `{"feature-1": [1, 3], "feature-2": [3, 4]}`. Let me fix that.

Reply:
The dictionary is more intuitive for me

Reply:
Sg. Should not be a problem. Just fixed, but let me re-run the test.

Reply:
Tested. Still works 🤔